### PR TITLE
PREFERENCES RESET [DANGER]

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

Resets our preferences to TG standard and transfers our changes. We were missing a few things that were causing runtimes. Also missed a chaplain PR.

Need one person to quickly read it over then we can merge it.

## Changelog
:cl:
refactor: The preferences system has been reset. Report any anomalies to github.
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/chaplain.dm [hunk 1]
```diff
@@ -74,57 +77,29 @@
...
-			new_bible = "Fluorescent Incandescence"
-		if("lol", "wtf", "gay", "penis", "ass", "poo", "badmin", "shitmin", "deadmin", "cock", "cocks", "meme", "memes")
-			new_bible = pick("Woodys Got Wood: The Aftermath", "War of the Cocks", "Sweet Bro and Hella Jef: Expanded Edition","F.A.T.A.L. Rulebook")
+		if("homosexuality", "gay", "penis", "ass", "cock", "cocks")
+			new_bible = pick("Guys Gone Wild","Coming Out of The Closet","War of Cocks")
+			switch(new_bible)
+				if("War of Cocks")
+					B.deity_name = pick("Dick Powers", "King Cock")
+				else
+					B.deity_name = pick("Gay Space Jesus", "Gandalf", "Dumbledore")
```
Comment (GoldenAlpharex) [inline code/modules/jobs/job_types/chaplain.dm:86]: What the fuck lmao

File: modular_skyrat/modules/customization/modules/client/preferences.dm [hunk 2]
```diff
@@ -192,7 +193,7 @@ GLOBAL_LIST_INIT(food, list(
 	/// If we have persistent scars enabled
 	var/persistent_scars = TRUE
 	///If we want to broadcast deadchat connect/disconnect messages
-	var/broadcast_login_logout = FALSE
+	var/broadcast_login_logout = TRUE
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/client/preferences.dm:196]: I mean, does that need to be on by default? Not that it matters, I'm just curious what the thoughts are behind it.

File: code/modules/jobs/job_types/chaplain.dm (hunk 1 above)
Comment (Gandalf2k15) [reply code/modules/jobs/job_types/chaplain.dm:86]: https://github.com/tgstation/tgstation/blob/master/code/modules/jobs/job_types/chaplain.dm

File: modular_skyrat/modules/customization/modules/client/preferences.dm (hunk 2 above)
Comment (Gandalf2k15) [reply modular_skyrat/modules/customization/modules/client/preferences.dm:196]: TG standard reset

File: modular_skyrat/modules/customization/modules/client/preferences_savefile.dm [hunk 3]
```diff
@@ -524,9 +537,6 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	jumpsuit_style = sanitize_inlist(jumpsuit_style, GLOB.jumpsuitlist, initial(jumpsuit_style))
 	uplink_spawn_loc = sanitize_inlist(uplink_spawn_loc, GLOB.uplink_spawn_loc_list, initial(uplink_spawn_loc))
 	playtime_reward_cloak = sanitize_integer(playtime_reward_cloak)
-	pda_style = sanitize_inlist(pda_style, GLOB.pda_styles, initial(pda_style))
-	pda_ringer = sanitize_text(pda_ringer, initial(pda_ringer))
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/client/preferences_savefile.dm:528]: You forgot to re-add that higher.

File: code/modules/jobs/job_types/chaplain.dm (hunk 1 above)
Comment (GoldenAlpharex) [reply code/modules/jobs/job_types/chaplain.dm:86]: Hahahaha

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/preferences_setup.dm [hunk 4]
```diff
@@ -41,14 +41,159 @@
 	if(randomise_flags & RANDOMIZE_FEATURES)
 		features = random_features()
 
-	//features = pref_species.get_random_features()
+	features = pref_species.get_random_features()
 	var/list/new_features = pref_species.get_random_features() //We do this to keep flavor text, genital sizes etc.
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/preferences_setup.dm]: It's already right under, I don't know why you're uncommenting it.
`features` doesn't seem to be used and gets overwritten by the following lines in the most part, so I'm not sure why this does any good.

**Comment by FlamingLily** [comment]
~~alt titles shiz got lost, reviewing now~~

wait nvm that's preferences_setup.dm not new_player.dm

File: modular_skyrat/modules/customization/modules/client/preferences_savefile.dm (hunk 3 above)
Comment (Gandalf2k15) [reply modular_skyrat/modules/customization/modules/client/preferences_savefile.dm:528]: It's there!

**Comment by hugmeorelse** [comment]
*chuckles* I'm in danger

**Pull request merged**